apps/web/test/lib/handleChildrenEventTypes.test.ts

```diff
@@ -1,4 +1,4 @@
-import prismaMock from "../../../../tests/libs/__mocks__/prisma";
+import prismaMock from "../../../../tests/libs/__mocks__/prismaMock";
```

Comment:
Let other users(other than bookingScenario) of prisma mock to keep on using prismaMock